### DISCUSSION TITLE
Could not update templates: failed to write templates

### DISCUSSION THREAD

**Discussion by LOONG001** [projectdiscovery/nuclei#2121] [General]
I have encountered strange problems when upgrading or unpacking templates on Windows systems, which fail even when using administrator privileges.
![pic1](https://user-images.githubusercontent.com/34571314/172288537-2d17224c-c03e-4dbb-990d-6ee9c33b9bd5.png)
![pic2](https://user-images.githubusercontent.com/34571314/172288544-ef9dba19-7492-4345-90cd-b7ff1e0e8e51.png)

**Comment by forgedhallpass** [comment]
Hello @LOONG001 

I've just cloned the repository on a Windows machine, and it works without any issue, hence I suspect the problem is with your system. Try deleting the project directory, or clone it to another folder. These actions should not require administrative privileges, if the initial clone was not done by an administrative account.
